src/dotnet-roslyn-tools/Products/FSharp.cs

```diff
@@ -0,0 +1,24 @@
+// Licensed to the.NET Foundation under one or more agreements.
+// The.NET Foundation licenses this file to you under the MIT license.
+// See the License.txt file in the project root for more information.
+
+namespace Microsoft.RoslynTools.Products;
+
+internal class FSharp : IProduct
```

Comment:
would we prefer these to be configurable via input parameters to the tool?  That way if anyone wants a new repo we can just have a new pipeline without needing to update the tool?

Reply:
these might have been already answered in previous reviews, let me know if so

Reply:
I think it makes sense that each command could have extra optional parameters to let people pass in the bits that command needs, rather than having to have a pre-defined product. Makes a lot of sense for calling these tools from a CI pipeline.

For using them on the command line directly, which is what I do for VS Mac insertions, having pre-defined products is really nice too :)

Reply:
I'm not against have other ways to specify a product. If you have a proposal, please open an issue and we can consider it. Personally, I hope more of the tooling becomes multi-product aware.

Reply:
So I just updated the code to do both - we have a set of default products (for more ease of use on command line), but instead users can also pass in their own arguments. Hopefully this approach makes everyone happy 😄

Reply:
Never! I refuse to be happy!